Changelog.md

```diff
@@ -22,6 +22,7 @@ Compiler Features:
 
 Bugfixes:
  * ABI decoder: Ensure that decoded arrays always point to distinct memory locations.
+ * Scanner: Fix doxygen-style parsing of multi-line comments with triple slashes when file is encoded with CRLF instead of LF.
```

Comment:
```suggestion
 * Scanner: Fix parsing of doxygen-style multi-line comments with triple slashes when file is encoded with CRLF instead of LF.
```

As counterproposal to the last suggestion - not sure in what way the parsing is "doxygen-style" ;-).

Reply:
Ah, wait. "Doxygen-style" is even incorrent, isn't it? Shouldn't it say natspec?

Reply:
Yeah, ``multi-line natspec comments`` is probably even better.